lib/assertions.js

```diff
@@ -1460,7 +1454,7 @@ module.exports = expect => {
     '<function> to [exhaustively] satisfy <function>',
     (expect, subject, value) => {
       expect.errorMode = 'bubble';
-      expect(subject, 'to equal', value);
+      expect(subject, 'to be', value);
```

Comment:
Using "to be" definitely describes the semantics much more accurately.

Reply:
It is also faster.